static/src/stylesheets/module/facia-garnett/_container--podcast.scss

```diff
@@ -93,8 +93,6 @@
     color: #ff6a56;
 }
 
-/* AB TEST WITH ACTUAL STORY IMAGE VS GENERIC */
-
 /* GENERIC IMAGE */
```

Comment:
This Generic Image code *should* be deletable too

Reply:
Isn't that there in case the card doesn't have a `card.displayElement`? e.g. https://github.com/guardian/frontend/blob/master/common/app/views/fragments/audio/containers/flagshipContainer.scala.html#L73

Reply:
Oh that's true, there is that case. That means we have to leave the CSS and the generic image lying around in case. What happens in a normal fronts container if the image fails for some reason? This is a hacky file but I guess we should try and replicate that

Reply:
I'm not sure whether it'll collapse or retain its dimensions. Can we test it by removing the image in browser devtools? Will the behaviour be specific to this element, or is it inherited from further up the cascade?